src/python-common/ceph/deployment/inventory.py

```diff
@@ -51,15 +51,15 @@ def __init__(self,
                  rejected_reasons=None,  # type: Optional[List[str]]
                  lvs=None,  # type: Optional[List[str]]
                  device_id=None,  # type: Optional[str]
-                 lsm_data={},  # type: Dict[str, Dict[str, str]]
+                 lsm_data=None,  # type: Optional[Dict[str, Dict[str, str]]]
```

Comment:
This is a nit pick for sure, but wouldn't it make sense to keep `lsm_data={}` if only for being able to drop `Optional[]` from the type signature? I don't see how this can ever be `None`